cmd/controller/app/controller.go

```diff
@@ -39,10 +40,16 @@ func Run(opts *options.ControllerOptions, stopCh <-chan struct{}) {
 	run := func(_ <-chan struct{}) {
 		var wg sync.WaitGroup
 		for n, fn := range controller.Known() {
+			// only run a controller if it's been enabled
+			if !util.Contains(opts.EnabledControllers, n) {
+				glog.Infof("%s controller is not in list of controllers to enable, so not enabling it", n)
+				continue
+			}
+
 			wg.Add(1)
 			go func(n string, fn controller.Interface) {
 				defer wg.Done()
-				glog.V(4).Infof("Starting %s controller", n)
+				glog.Infof("Starting %s controller", n)
```

Comment:
I've set the logging of controllers starting up to a higher log level so that it's easier to see what's going on in case someone turns a controller off by mistake